.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "java.compile.nullAnalysis.mode": "automatic",
+    "editor.tabSize": 2,
+    "java.configuration.updateBuildConfiguration": "interactive"
+}
```

Comment:
This content is specific to vscode right? Could we delete this, or add it to `.gitignore`